public/locally-available-studies.json

```diff
@@ -1,24 +1,31 @@
 [
   {
-    "name": "Rally Base Study",
+    "name": "Your Time Online and \"Doomscrolling\"",
```

Comment:
Mh, why are we changing this? It feels weird to do integration testing on a live study. This will end up sending bogus data .

Reply:
@Dexterp37 It shouldn't send data because the core add-on will be in dev mode with sending data disabled, right? Happy to revert this.

Reply:
I'm pretty sure we'll need to update the Basic Study (which is on AMO) to update `rally.js`, I was hoping we did not need to keep this "study" but if it's necessary then we can do so.

Maybe the real fix, though, is to use a synthetically-generated study for integration tests?

Reply:
Just to be clear though our integration tests *shouldn't* be sending any data anywhere now, and I don't think changing this would change that.

I can actually go ahead and revert this, the reason we got here is a little complex but here it is for posterity:

1. we're maintaining this "Rally Base Study" on AMO just for testing purposes, while this has some benefit from a testing PoV it's using the network and takes a long time to update (add-on review)
2. I thought we'd need to update (1) but since I ended up using remote-settings it's not necessary after all
3. I don't think we should be sending data from the integration tests whether this is changed or not, that's configured by the core add-on

Reply:
> Maybe the real fix, though, is to use a synthetically-generated study for integration tests?

Yes, the real fix is generating a study on the fly (we have the means to do that).